Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorHelpers.h

```diff
@@ -62,5 +60,6 @@ namespace AzToolsFramework
 
     private:
         const EditorVisibleEntityDataCache* m_entityDataCache = nullptr; //!< Entity Data queried by the EditorHelpers.
+        FocusModeInterface* m_focusModeInterface = nullptr;
```

Comment:
suggestion: Hopefully `IsInFocusSubTree` is const so we could also make this pointer `const` too (it would be nice if we could guarantee that nothing changes the entities from this 'view' type class (until we have to that is, which might happen)